mantidproject/mantid
# A bug in FunctionBrowser's tie setting

Update the function index property. Corrected tie string format.

**To test:**

* Open the multi data fit browser.
* Add a function.
* Fix or tie a parameter.
* Add another function.
* Check that there is no error and the tie remains set.

Fixes #19615.

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
